test/main.js

```diff
@@ -24,6 +26,8 @@ var SelfInsertingView = View.extend({
     render: function () {}
 });
 
+viewCompliance.view(test, ViewSwitcher, { view: new makeTestView()() });
```

Comment:
They were passing locally for me.  I ran the tests through the console and with the browser UI. When I get a chance I will run them again to confirm.